azure/basic-aks-cluster/README.md

```diff
@@ -10,25 +12,42 @@ If you're new to XebiaLabs blueprints, check out:
 * [Get started with blueprints](https://docs.xebialabs.com/xl-platform/concept/get-started-with-blueprints.html)
 * [Get started with XL JetPack](https://docs.xebialabs.com/xl-platform/concept/get-started-with-xl-jetpack.html)
 
-## Prerequisites
+## Usage
+
+To use this blueprint, run `xl blueprint` in an empty directory and select:
+
+```plain
+azure/basic-aks
+```
+
+## Tools and technologies
+<!--Dev create this section for consistency with other blueprint readmes (see https://github.com/xebialabs/blueprints/tree/master/gcp/microservice-ecommerce for example)-->
+
+## Minimum required versions
 
-### Authenticate to Azure
+This blueprint version requires at least the below versions of the specified tools to work properly.
+
+- XL Deploy: Version 8.6.1
+- XL CLI: Version 8.6
 
-> Note: This blueprint requires your Azure login credentials in order to create the cluster.
+## Prerequisites
+To run the YAML that this blueprint generates, you need:
+* Azure login credentials in order to create the cluster
+* [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/). Ensure that you are [signed in](https://docs.microsoft.com/en-us/cli/azure/authenticate-azure-cli?view=azure-cli-latest).
```

Comment:
Maybe be more specific... "signed in to Azure" (as opposed to a XebiaLabs product)

Reply:
Will do